api/init.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	"sync"
+
+	"github.com/raintank/metrictank/expr"
+	"gopkg.in/raintank/schema.v1"
+)
+
+const defaultPointSliceSize = 2000
```

Comment:
we currently keep meters of the number of series per request and number of points per request (reqRenderSeriesCount, reqRenderPointsFetched).  I think we should also keep counters of pointsFetched and number of series fetched.

the current metrics give us good insight into query complexity, but the counters would provide a good measure of overall request workload.

  As well as providing a good measure of query workload we can then also compute average points per series per request, among other things.